src/react/components/pages/homepage/homePage.tsx

```diff
@@ -144,6 +144,10 @@ export default class HomePage extends React.Component<IHomepageProps> {
     }
 
     private deleteProject = async (project: IProject) => {
-        await this.props.actions.deleteProject(project);
+        try {
+            await this.props.actions.deleteProject(project);
+        } catch (error) {
+            throw new AppError(ErrorCode.ProjectDeleteError, "Error deleting project file");
```

Comment:
what happens if project file is gone but it's still showing in `Recent Projects`?  Is there a force delete option?

Reply:
I was also thinking that as well.  I didn't want to complicate this story with too much extra but i will add an item on the backlog to address this.

Reply:
AB#17063